test/integration/src/XcooBee/Core/Api/InboxTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Test\XcooBee\Core\Api;
+
+use XcooBee\Test\IntegrationTestCase;
+
+class InboxTest extends IntegrationTestCase
+{
+
+    public function testInboxApi()
+    {
+        $inboxApi = self::$xcoobee->inbox;
+        $inboxData = $inboxApi->listInbox();
+        if (isset($inboxData->result->inbox->data[0])) {
```

Comment:
can we divide this tests to different cases? Your inbox should be added in `BeesTest`
First `list`
Second `get`
Third `delete`. In this one check if file was removed (call `list`)

And we can check some result data here. Not only status code